packages/docs/versioned_docs/version-2.0.0-alpha-3/protocol/04-reputation.md

```diff
@@ -4,13 +4,15 @@ description: Definition of reputation in UniRep
 
 # Reputation
 
-## Reputation
+Attesters define the reputation system for their application on top of the UniRep reputation standard.
 
-* The reputation in UniRep protocol includes
-  * `posRep`: is the positive reputation given by the attester
-  * `negRep`: is the negative reputation given by the attester
-  * `graffiti`: is the message given by the attester
-  * `timestamp`: is the timestamp the last graffiti was received
+* TODO: describe purpose of using + and - rep values
```

Comment:
I remember seeing somewhere a mention of the reason for this implementation but now I can't find it. I think it would be relevant here.

Reply:
By storing positive and negative reputation separately we can represent net negative reputation values without using signed integers in contracts/circuits.

e.g. a user has `-10` reputation. We can represent this using positive values only by saying they have `100` positive reputation and `110` negative reputation.